pandas_flavor/register.py

```diff
@@ -1,5 +1,85 @@
 from functools import wraps
-from pandas.api.extensions import register_series_accessor, register_dataframe_accessor
+from pandas.api.extensions import (
+    register_series_accessor,
+    register_dataframe_accessor,
+)
+import inspect
+from contextlib import nullcontext
+
+method_call_ctx_factory = None
+
+
+def handle_pandas_extention_call(method, method_signature, obj, args, kwargs):
```

Comment:
Function spelling change.

```suggestion
def handle_pandas_extension_call(method, method_signature, obj, args, kwargs):
```